libcontainer/cgroups/fs2/cpu.go

```diff
@@ -29,15 +33,56 @@ func setCpu(dirPath string, cgroup *configs.Cgroup) error {
 		}
 	}
 
-	// NOTE: .CpuQuota and .CpuPeriod are not used here. Conversion is the caller's responsibility.
-	if cgroup.Resources.CpuMax != "" {
-		if err := fscommon.WriteFile(dirPath, "cpu.max", cgroup.Resources.CpuMax); err != nil {
+	// Convert .CpuQuota and .CpuPeriod into "cpu.max".
+	// Conversion requires the previous value of "cpu.max"
+	prevCpuMaxBytes, err := ioutil.ReadFile(filepath.Join(dirPath, "cpu.max"))
```

Comment:
We should not do anything if quota and period is not set. This read is not needed in this case.